src/index.js

```diff
@@ -41,6 +42,7 @@ export default class SuperImage extends React.Component {
     width       : null,
     height      : null,
     alt         : '',
+    role        : null,
```

Comment:
@tsuyoshiwada @masuP9 I think that it may be the default `role : 'img'`. How about you?

Reply:
Hi @tommy-san, Thank you for reviewed !!

Since `<img />` itself plays the same role as `role="img"` by default, I think that it is unnecessary to specify explicitly :bulb:

Reply:
I forgot `<img {...props} />` 😢 
It looks good with your PR.
thanks!

Reply:
Thank you, too :smile: !